### PR TITLE
Add LED Strip sliders

### PR DESCRIPTION
**Pull request by ASDosjani**
Resubmitting https://github.com/betaflight/betaflight-configurator/pull/3531 with MSP2 commands and semver check.

REQUIRES Betaflight PR: https://github.com/betaflight/betaflight/pull/12995

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Comment by nerdCopter** [comment]
- smart use of semver :brain: :heart_eyes: 
- UI testing is on my to-do list (no hardware though)

**Comment by nerdCopter** [comment]
* `230af095` seems to work for UI portion. 
* tested un-soldered foxeerf405 with firmware 4.4.3 and with PR #12995, both acted as expected.

**Review by haslinghuis — APPROVED** [review]

**Review by nerdCopter — APPROVED** [review]

**Pull request merged**
